build.xml

```diff
@@ -478,6 +481,35 @@
         </exec>
     </target>
 
+    <!-- Include annotation processor for VirtualTable -->
+    <property name="processor.build.classes" value="${build.classes}/processor" />
+    <property name="processor.manifest" value="${processor.build.classes}/MANIFEST.MF" />
+
+    <target name="annotation-processor-build" depends="init" description="Build system view annotation processor">
+        <antcall target="_annotation_processor_build"/>
+    </target>
+
+    <target name="_annotation_processor_build">
+        <mkdir dir="${processor.build.classes}" />
+        <javac includes="**/virtual/proc/*.java"
+               destdir="${processor.build.classes}"
+               includeantruntime="true"
+               source="${java.default}" target="${java.default}">
+            <src path="${build.src}"/>
+        </javac>
+    </target>
+
+    <target name="annotation-processor-jar"
+            depends="annotation-processor-build">
+        <mkdir dir="${processor.build.classes}/META-INF" />
+        <jar destfile="${build.dir.lib}/jars/cassandra-annotation-processor-${version}.jar">
+            <fileset dir="${processor.build.classes}"/>
+            <manifest>
+                <attribute name="Built-By" value="Maxim Muzafarov"/>
```

Comment:
is this needed ?

Reply:
I did it the same way with the other stuff in the build.xml, but yeah it looks weird. Fixed.